src/component/App/App.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { HashRouter as Router, Route } from 'react-router-dom';
+
+import CountryInformation from '../CountryInformation/CountryInformation';
+import СountriesList from '../CountriesList/CountriesList';
+import Footer from '../Footer/Footer';
+import GlobalStyle from './GlobalStyle';
```

Comment:
prefer `Named` over `Default` imports/exports 

`import { Footer } from '../Footer/Footer';` (index.ts)

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Reply:
Oh! indeed. The secrets you never ever put in the repo. Should exists a really big reason why you would do that. You may use something like a config file as pointed @shurygindv. A good tool would be https://www.npmjs.com/package/dotenv